steam_wrapper/steam_wrapper.py

```diff
@@ -295,7 +295,10 @@ def get_current_xdg_dir_prefix():
     if not os.path.isdir(os.path.expanduser("~/.steam")):
```

Comment:
Instead of catching readlink failure below, I'd rather do check the `~/.steam/root` symlink here. Something like this:
```suggestion
    steam_root_link = os.path.expanduser("~/.steam/root")
    if not (os.path.islink(steam_root_link) and os.path.isdir(steam_root_link)):
```

Reply:
Makes sense.